# 1.Order of Magnitudes

## Acceleration

|Process|Acceleration|
|-|-|
|cell fractionation by centrifugation|1000 to 150,000 times gravity|

## Speed

|Process|Speed|
|-------|-----|
|DNA Replication|bacteria: 1000 nuleotides/second<br>eukaryotes: 50 nucleotides/second|
|each RNA polymerase|50 nucleotides/second|
|Dividing of *E.coli*|once/30 minutes|
|action potential|100 m/s|




## Length

|Item|Length|
|-|-|
|nucleus|5 $\mu$m|
|a bacterial cell|$1\ \mu \text{m}$|
|a ribosome|20 nm|
|membrane thickness|5 nm|
|vesicle (liposome) diameter|25 nm ~ 1 μm|
|virus|100 nm|
|a frog egg|1mm|
|skin cell|30 μm|
|red blood cell|8 μm|
|Leading Strand|$\infty$|
|Lagging Strand|1000~2000 nucleotides for bacteria<br>100~200 nucleotides for eukaryotes|
|RNA primers|10 nucleotides for eukaryotes|
|Intervals between RNA primers on lagging strand|100~200 nucleotides|
|RPA (domains of SSB) covers|8 nucleotides|
|genome of *E. coli*|$4.6\times10^6$ nucleotide pairs|
|average-size human chromosome|$150\times10^6$ nucleotides|
|mitochondrion|length: $1\sim2\mu \mathrm{m}$<br>width: $0.1\sim 0.5 \mu \mathrm{m}$|
|TATA box to transcription start site|30 nucleotides upstream|
|CTD of RNA polymerase|$52\times 7$ amino acids|
|DNA probe|30 nucleotides|



## Rate

|Process|Rate|
|-|-|
|DNA Replication|error rate: $\frac{1}{10^{10}}$|
|DNA Bases Tautomer| $\frac{1}{10^{4}}$~$\frac{1}{10^{5}}$|
|5'$\rightarrow$ 3' polymerization(DNA)|error rate: $\frac{1}{10^5}$|
|3'$\rightarrow$ 5' exonucleolytic proofreading(DNA)|error rate: $\frac{1}{10^2}$|
|strand-directed mismatch repair(DNA)|error rate: $\frac{1}{10^3}$|
|RNA polymerase|error rate: $\frac{1}{10^4}$|



## Ratio

|Substances|Ratio|
|-|-|
|ATP : dATP|500 : 1|
|mRNA : RNA|3~5 : 100|
|cholesterol : lipid bilayer<br>in weight in animal cells |20 : 100|
|protein : plasma membrane<br> in mass|1 : 2|
|protein : lipid<br>in number|1 : 50|
|$c(Na^+)_{extra}：c(Na^+)_{cytosol}$|10$\sim$ 30 : 1|
|$c(K^+)_{cytosol}：c(K^+)_{extra}$|10$\sim$ 30 : 1|


## Number

|Object|Number|
|-|-|
|Species on the planet|$10^8$|
|replication origin|bacteria: 1<br>eukaryote: 30,000~50,000|
|unicellular bacterium|500 genes|
|*E.coli*|4,300 genes|
|a human|25,000 genes|
|types of mRNA in each cell|10~15|
|Human fibroblasts|divide 25~40 times in culture|
|average division time of a normal cell<br>Hayflick limit|50~70|
## Duration

|Process|Duration|
|-|-|
|DNA Synthesis phase<br>S phase|mammalian cell: 8 hours<br>yeast: 40 minutes|
|$Na^+$ pumping cycle|10 ms|

## Time

|Item|Time|
|-|-|
|ancestral cells|$3.5\sim 3.8 \times 10^6$ years ago|








# 2.Protein and its Function
## DNA Replication

|Protein|Function|
|-|-|
|DNA polymerase|1.free deoxyribonuleoside triphosphates $\rightarrow$ DNA<br>2.proof reading|
|*3'-to-5' proof reading exonuclease* <br>(a subunit or protein domain of DNA polymerase)|self-correcting|
|Polymerase ε|synthesize leading strand in eukaryotes|
|Polymerase α<br>Polymerase δ|synthesize lagging strand in eukaryotes<br>Polα: RNA primer+ 20 n.t.;<br>Polδ: 200n.t.; mismatch repair|
|DNA primase|free ribonuleoside triphosphates $\rightarrow$ RNA primers <br>on the lagging strand|
|DNA ligase|joins the 3' and 5' ends of DNA|
|DNA helicase|5' to 3' along template strand in bacteria|
|DNA-binding proteins<br>(Single Strand+..., SSB)|prevent hairpin formation|
|RPA(Replication Protein A):<br>two DNA binding domains of SSB||
|Sliding Clamp|holds DNA polymerase onto DNA|
|Clamp loader|load clamp onto DNA|
|DNA topoisomerase:<br>a reversible nulease|type I: single strand break<br> type II: double strand break|
|Initiator protein||
|Helicase loading protein||
|ORC|origin recognition complex|
|telomerase|maintain the telomeres|

## RNA Replication
Especially in RNA virus like SARS-CoV-2.
|Protein|Function|
|-|-|
|replicase<br>RNA dependent RNA polymerase|from RNA to RNA|

## DNA Transcription

|Protein|Function|
|-|-|
|RNA polymerase holoenzyme<br>(bacteria)|1.unwinding DNA helix<br>2.free ribonuleoside triphosphates $\rightarrow$ RNA|
|σ factor<br>(bacteria)|binds to promoter<br>to begin transcription|
|core enzyme<br>(bacteria)||
|RNA polymerase I<br>(eukaryote)|rRNA|
|RNA polymerase II<br>(eukaryote)|mRNA, miRNA, siRNA|
|RNA polymerase III<br>(eukaryote)|tRNA, rRNA|
|general transcription factors<br>(eukaryote)<br>TFIIA, TFIIB, TFIID...|to begin transcription|
|TBP<br>TATA-binding protein|a subunit of TFIID to bind TATA box|
|CTD<br>C-terminal domain, tail of RNA polymerase|be phosphorylated by TFIIH<br>make polymerase disengage from general transcription factors|
|regulator|activator and repressor|
|activator|bind to enhancer to attract general transcription factors and RNA polymerase II|
|repressor or<br>inhibitor|bind to operator to impede RNA polymerase binding|
|mediator|1.allow activator communicate with RNA polymeraseII<br>2.position TFIIH near CTD|
|CAP|an activator work with cAMP|
|elongation factors|keep RNA polymerase elongating|
|DNA topoisomerase<br>called DNA gyrase in bacteria||
|reverse transcriptase enzyme|from mRNA to cDNA|

### About Signal

|Enzyme|Function|
|-|-|
|kinase<br>related to ATP|phosphotransferase<br>transfer phosphate from organic compounds like ATP<br>or transfer phosphate to ADP to produce ATP|
|phosphorylase<br>unrelated to ATP|adding inorganic phosphate|
|phosphatase<br>unrelated to ATP|dephosphorylation, unrelated to ATP|
|calmodulin|relay cytosolic $Ca^{2+}$signal|
|calmodulin dependent kinases<br>CaM-kinases|autophosphorylation<br> cell memory effect|
|cyclic-AMP-dependent protein kinase<br>protein kinase A<br>PKA|relay cAMP signal|
|heterotrimeric GTP-binding protein<br>G protein|relay signal from GPCR|
|adenylyl cyclase|synthesize cAMP from ATP|
|cAMP phosphodiesterase|hydrolyze cAMP|
|G-protein-coupled receptors<br>GPCR|cell surface receptor|

### Molecular Motors
|Protein|Function|
|-|-|
|kinesin||
|dynein||

## Homologous Recombination (HR)

|Protein|Function|
|-|-|
|RecA|*E. coli* Homologous recombination<br>searching and strand exchange|
|Rad51|eukaryotes Homologous recombination<br>searching and strand exchange|

## ATP Related Process

|Protein|Function|
|-|-|
|V-type ATPase<br>V stands for vacuolar<br>$V_o$ integral domain and $V_1$ peripheral domain|acidifying intracellular compartments|
||
|F-type ATPase<br>ATP synthase|ADP $\rightarrow$ ATP<br>oxidative phosphorylation <br>photophosphorylation|

## Enzyme Kinetics
* To study the mechanism of purifies enzymes:
  * 3-D structure
  * site-directed mutagenesis
  * rate of a reaction and its response to changing parameters (enzyme kinetics)
  

# 3.RNA

|Acronym|denotation|
|-|-|
|mRNA|messenger RNA|
||noncoding RNAs|
|rRNA|ribosomal RNA|
|tRNA|transfer RNA|
|snRNA|small nuclear RNA|
|miRNA|microRNA|
|siRNA|small interfering RNA|
|piRNAs|piwi-interacting RNAs|
|lncRNAs|long noncoding RNAs|
||Telomerase RNA|



# 4.Position of the DNA

|Position|Function|
|-|-|
|replication origins|first be opened|
|transcription unit|transcribed segment of DNA|
|promoter|binding RNA polymerase|
|TATA box|in promoter to bind TBP, a subunit of TFIID|
|enhancer|binding activator|
|cis-regulatory DNA sequence|operator and enhancer|
|operon|operated by binding of operator and repressor|
|telomere|ends of DNA|


# 5.Complex

* The DNA Replication Fork
* Transcription initiation complex/machinery
* RNA-processing machinery

# 6.Signal

## Phosphorylation

|Position|Effect|
|-|-|
|Ser5 in CTD of RNA polymerase|1.polymerase disengages from general transcription factors<br>2.RNA-processing machinery load onto the polymerase|


# 7.Disease

|Cause|Disease|
|-|-|
|deficient mismatch proofreading|a type of colon cancer: HNPCC|

# 8.Techniques
## Microscopy

|Technoques|Applications|
|-|-|
|autoradiography|eukaryotic chromosome replication:<br>rate and direction|
|Atomic Force Microscope<br>AFM|get 3D contour map of the surface<br>resolution:<br>0.1 nm in the vertical dimension<br>0.5~1.0 nm in the lateral dimension|
|transmission electron microscope<br>TEM||
|scanning electron microscope<br>SEM|look at the surface details of cells (3D)|
|cryo-electron microscopy|atomic level|
|X-ray crystallography|atomic level|
|fluorescent microscope||
|confocal microscope<br>a specialized fluorescent microscope|3D imaging<br>150 μm in depth|
|FRET<br>Fluorescence Resonance Energy Transfer|distance and interactions between molecules|
|Biosensor<br>FRET as reporting parts combined with sensing parts||
|FRAP<br>Fluorescence Recovery After Photobleaching|detect protein's kinetic parameters|
|SPT<br>single-particle tracking microscopy|relies on antibody-coated gold nanoparticles tagged with protein|
|interference contrast optics<br>DIC|distinguish the internal structure of a cell|
|phase contrast optics||
|computed tomography<br>CT<br>and deconvolution|3D<br>40 μm in depth|
|Structured Illumination Microscopy<br>SIM|100 nm|
|Stimulated Emission Depletion microscopy<br>STED|20 nm<br>relies on photoswitchable dyes or proteins|
|Single-Molecule Localization Microscopy<br>SMLM<br>Photoactivated Localization Microscopy<br>PALM<br>Stochastic Optical Reconstruction Microscopy<br>STORM|20 nm<br>relies on photoswitchable dyes or proteins|
|Expansion Microscopy<br>ExM|detect mRNA|
|light sheet microscopy|low photobleaching and photodamage<br>3D|
|total internal reflection fluorescence microscopy<br>TIRF|0.2 μm in depth|
|2D polyacrylamide gel electrophoresis|separate proteins according to size and charge|

## Recombinant DNA Technology

|Term|Denotation|
|-|-|
|Genomic DNA Library<br>Genomic Clone|From Genome in Bacterial Plasmid<br>with introns and extrons|
|cDNA Library<br>cDNA Clone|From mRNA in Bacterial Plasmid<br>suitable for synthesize protein artificially|
|Ploymerase Chain Reaction<br>PCR|when complete genome is known<br>under 10,000 nucleotide pairs|
|DNA denaturation|90 $^\circ$C|
|DNA renaturation (hybridization)||
|DNA probe||
|Fluorescence in situ Hybridization<br>FISH||
|Gibson assembly|join DNA and plasmid|
|oligonuleotides|as primer or probe|
|site-directed mutagenesis|by PCR|
|restriction nuclease||
|agarose gel|>500 nucleotides|
|polyacrilamide gel|<500 nucleotides|
|oulsed-field gel|chromosome, up to millions nucleotides|
|bacterial artificial chromosome<br>BAC|maintain very long DNA, up to 1 million nucleotides|

## Plasmid and its Components

|Gene|Protein|Chemical|Function|
|-|-|-|-|
|Lac I|Lac repressor||express Lac inhibitor (repressor)|
|||ampicillin|antibiotic|
|||kanamycin|antibiotic|
|||tetracycline|antibiotic|
||T7 polymerase||as RNA polymerase to transcribe|
|KanR|||kanamysin resistant gene, encoding a phosphotransferase to destroy kanamysin|
|||IPTG|lactose analogs to bind Lac repressor and therefore activate lac operon transcription|
|||X-gel|lactose analogs|
|Lac Z|galactosidase|||
|ROP|||regulate plasmid copy number|

### Other Related Terms and its Meanings

|Term|Meaning|Function|
|-|-|-|
|OD600|the optical density of a sample measured at a wavelength of 600 nm in 1 cm light path|measure cell population|
|SDS|Sodium Dodecyl Sulfate|denature protein<br>disrupt cell walls|
|PBS|Phosphate-buffered saline |a buffer solution (pH ~ 7.4) |

## Cell Engineering

|Term|Meaning|
|-|-|
|primary culture|directly from tissue|
|embryonic stem cell|
|pluripotent|
|callus|undifferentiated plant tissue|
|replicative cell sequence|
|immortalized cell line|
|culture shock|a protective mechanism to stop division|
|transformed cell line|
|B lymphocyte|
|hybridoma|
|monoclonal antibody|
|organoid|


## Purifying Proteins

|Term|Meaning or Purpose|
|-|-|
|slurry or homogenate or extract|
|preparatice ultracentrifuge|separate cell components by size and (buoyant) density|
|centrifugation|1.nuclei sediment<br>2.mitochondria<br>3.vesicles<br>ribosomes<br>salt solution|
|velocity sedimentation<br>differential centrifugation|shallow gradient|
|equilibrium sedimentation<br>densty gradient centrifugation|steep gradient|
|density gradient<br>sucrose+cesium chloride gradient|prevent convective mixing from distorting separation|
|S value<br>sedimentation coefficient|
|column chromatography|1.ion-exchange chromatography<br>2.hydrophobic chromatography<br>3.gel-filtration chromatography<br>4.affinity chromatography e.g. immunoaffinity chromatography and metal affinity chromatography|
|High-performance liquid chromatography<br>HPLC||
|immunoprecipitation|antibody-coated beads is added to a protein extract|
|epitope|antigenic determinant|
|tandem affinity purification tagging<br>TAP-tagging|two step strategy|
|purified cell free system|

## Analyzing Proteins

|Term|Meaning or Function|
|-|-|
|sodium dodecyl sulfate polyacrylamide-gel electrophoresis<br>SDS-PAGE|1.unfold polypeptide chain<br>2.negative charge detergent<br>3.mass and subunit composition of protein|
|Coomassie blue|the dye staining proteins|
|2D gel electrophoresis|separated by<br>1.intrinsic charge (isoeletric point)<br>2.size|
|isoelectric focusing|
|SDS-PAGE|
|Western blotting<br>immunoblotting|
|hydrodynamic measurements|
|mass spectrometry|
|ion source|
|matrix-assisted laser desorption ionization<br>MALDI|
|mass analyzer|
|detector|





## DNA Sequencing

|Method|Mechanism|
|-|-|
|dideoxy sequencing<br>Sanger sequencing|ddNTP as chain terminating nucleotides|
|Illumina sequencing<br>2nd generation|reversible termination one by one|
|single molecule realtime sequencing<br>SMRT<br>3rd generation|by fluorescent dye attached dNTPs|
|nanopore sequencing<br>3rd generation|by electric current difference caused by different nucleotides passing the nanopore|
|deep RNA sequencing<br>RNA-seq|entire repertoire of RNA<br>valuable tool for annotating genomes in different cells|
|open reading frames<br>ORFs|DNA to codons|

## Others

|Term|Meaning|
|-|-|
|microarray|detect gene expression|
|RNA sequencing|








# 9.Models

* Lac repressor
  * $\text{strength of repression} = f(\text{distance between two binding sites})$

* *E.coli*
  * $\text{growth rate} = f(\text{initial buoyant mass})$
  * Binomial partitioning model of
mRNA and proteins

* ATP synthase
  * electron transfer and ATP synthesis are obligately coupled
    * chemiosmotic model
    * proton motive force
# 10.Components of the Cell

## Membrance Structure

|Term|Meaning or Function|
|-|-|
|plasma membrane|1.communication<br>2.import and export molecules<br>3.growth and motility|
|internal membrane|
|lipid bilayer|
|amphipathic|a molecules with both hydrophilic and hydrophobic parts|
|phospholipid|
|phosphatidylcholine|a commom type of phospholipid|
|hydrophilic head|choline and phosphate group|
|hydrophobic tail|fatty acid|
|cholesterol|make the membrane less flexible and less permeable|
|glycolipid|sugar as hydrophilic head|
|endoplasmic reticulum<br>ER|manufacture phospholipid|
|scramblase|a type of transporter protein<br>remove phospholipid from one half and insert them in the other randomly|
|Golgi apparatus|1.cause asymmetry<br>2.add sugar to glycolipid|
|flippase|transfer **specific** phospholipid from one side to the other, consuming ATP|
|membrane proteins|
|glycoprotein|protein + oligosaccharide|
|proteoglycan|protein + polysaccharide|
|transporter|
|ion channel|
|anchor|
|receptor|
|enzyme|
|transmembrane protein|belongs to integral membrane protein|
|lipid -linked protein|belongs to integral membrane protein|
|monolayer attached protein|belongs to integral membrane protein|
|intergral membrane prorein|can be removed only by disrupting the bilayer using detergent|
|peripheral membrane protein| can be removed by gentle extraction procedures|
|protein-attached|belongs to peripheral membrane protein<br>attached to membrane protein instead of lipid|
|single-pass membrane protein|
|multipass membrane protein|
|β barrel|
|α helix|
|detergent|SDS|
|micelle|small irregularly shaped cluster|
|bacteriorhodopsin|a transmembrane pump|
|retinal|
|cell wall|
|extra cellular matrix|
|cell cortex|specialized inner layer in animal cytoplasma governing shape and movement|
|spectrin|dimeric protein forming cell cortex in red blood cells|
|attachment protein|
|actin|
|myocin|motor protein|
|membrane domain|funtionally specialized region|
|tight junction| a kind of cell junctions|
|oligosaccharide|
|carbohydrate layer<br>or<br>glycocalyx|protect, lubricate, recognition, adhesion|
|lectin|transmembrane protein specialized to bind oligosaccharide chains (glycolipid) on endothelial cells|
|neutrophils|a type of white blood cell|

## Membrane Function

|Term|Meaning|Supplements|
|-|-|-|
|membrane transport protein|
|transporter|by changing shape|
|(ion) channel|pass by diffusion|
|facilitated transport|
|simple diffusion|
|cell respiration|
|membrane potential|
|resting membrane potential|
|passive transport|
|active transport|
|electrochemical potential|
|osmosis|water moves down its concentration gratient|
|aquaporin|specialized channels for water|
|ruegor pressure|
|glucose transporter||passive transport<br>D-glucose is used as energy|
|glucose||produce by glycogen|
|pumps|gradient-driven;<br>ATD-driven;<br>light-driven|
|ATP-driven $Na^+$ pump||$Na^+-K^+$ ATPase<br>$Na^+-K^+$pump|
|ATP-driven $Ca^{2+}$ pump|
|symport|
|antiport|
|uniport|
|$Na^+-H^+$ exchanger||antiport|

## Connections of Terms
* glycocalyx
  * glycoprotein
  * proteoglycan
  * glycolipids
* nervous system
  * neuron
    * axon
      * ion channel
        * Na voltage-gated channel
        * K leak channe
      * transporter / pumps
        * Na pumps
      * membrane potential
        * Nernst Equattion
        * depolarization
        * action potential
        * resting membrane potential
    * dendrite
  * synapse
    * neurotransmitter
      * excitaory neurotransmitter
        * transmitter-gated Na channel
      * inhibitory neurotransmitter
        * transmitter-gated Cl channel
    * ion channel
      * ligand-gated channel
      * transmitter-gatet ion channel
    * synaptic vesicles
* transport / membrane transport protein
  * active trnasport
    * transporter / pump
      * Na Ka
      * Ca
      * H
  * passive transport
    * electrochemical gradient
      * Nernst Equation
    * ion channel
      * selectivity filter
      * gradient driven channel
        * symport
        * antiport
      * light-gated
      * mechanically-gated
      * voltage-gated
        * Na
      * ligand-gated
        * trasmitter-gated
      * other typical channel
        * K leak channel
        * aquaporin
    * transporter
* osmosis
  * turgor pressure
* technology
  * patch-clamp recording


## Recognition Mechanism
* **transport vesicle** to target membrane
  * target membrane
    * t-SNARES
    * tethering protein
  * transport vesicle
    * v-SNARES
    * Rab protein
* **ribosome** to ER membrane
  * ER membrane
    * SRP receptor
    * protein translocator
  * intermediary
    * SRP signal recognition particle
  * ribosome
    * signal sequence
* **clathrin** to vesicles
  * vesicle
    * adaptin receptor
  * intermediary
    * adaptin
  * clathrin
* **cholesterol** *through* cell membrane
  * cell membrane
    * LDL receptor
  * intermediary
    * LDL low density lipoprotein
      * bind to cholesterol in blood
      * release cholesterol in endosome
      * LDL itself is degraded in endosome
  * cholesterol
* **Iron** *through* cell membrane
  * cell membrane
    * transferrin receptor
  * intermediary
    * transferrin
      * bind to iron at neutral pH in blood
      * release iron at acidic pH in endosome
      * recycle of transferrin by pH *difference*
  * iron
* **nuclear proteins** *through* nuclear pores
  * nuclear pores
    * NPC nuclear pore complex meshwork
    * nuclear bascket
  * intermediary
    * nuclear import receptor (bind to cargo)
    * recycle of import receptor by Ran-GTP and Ran-GDP
    * Ran-GTP (release cargo)
    * recycle of Ran-GTP by 
      * Ran-GAP (GTPase-acitvating protein)
      * Ran-GEF (GTP exchange factor)
  * nuclear proteins 
    * nuclear locolization signal
  
> **Comments**:
> When the cell wants to transport C(cargo) through B(barrier) there must be a pair of opposite processes,
> one of them load the cargo while the other one release the cargo.
> That reflects a reversible binding process, which must be carried out at different conditions.
> For example the pH differences and the existence of Ran-GAP and Ran-GTP.
> Meanwhile, how to maintain that difference is another problem including the proton pump and recycle of Ran-GTP.

### Interaction with Small Particles
* Molecular Oxygen
  * In cytochrome c oxidase, oxidative phosphorylation
    * Cu (copper)
    * heme group including Fe (iron)
  * In water-splitting enzyme, photosystem II 
    * Mn (manganese) cluster
* Photon
  * photosystem (photosynthesis)
    * antenna complex
      * chlorophyll II including Mg (magnesium)
    * chlorophyll II *sepcial pair*
  * rhodopsin
    * retinol & retinal
  * bacteria rhodopsin: light-driven proton pump







